tests/__init__.py

```diff
@@ -163,3 +164,13 @@ def suspend_capture_stdin(pytestconfig: Any) -> Iterator[None]:
 
     if pytestconfig.getoption("capture") == "no":
         capmanager.resume_global_capture()
+
+
+@contextlib.contextmanager
+def ci_group(s: str) -> Iterator[None]:
+    if os.getenv("MKOSI_CI_SECTION") == "github":
```

Comment:
Just check for `$GITHUB_ACTIONS` here which is always set to true when running in github actions.